modules/ROOT/pages/for-each-scope-concept.adoc

```diff
@@ -45,7 +45,11 @@ This is an example XML based on the For Each scope configuration detailed above:
 
 == Variable Propagation
 
-Every execution of the For Each scope starts with the variables and values from the previous execution of the block. New variables or modifications to existing variables that take place when processing one element are visible during the processing of another element. However, these changes to variables will not be available outside the For Each scope. In other words, the set of variables (and their values) after the execution of the For Each Scope remain the same as before its execution.
+Every execution of the For Each scope starts with the variables and values
+from the previous execution of the block. New variables or modifications
+to existing variables that take place when processing one element are visible
+during the processing of another element. However, these changes to variables
+will not be available outside the For Each scope.
```

Comment:
The changes are available outside the for each scope

Reply:
Also, example below must be updated to: 
```
After aggregation, the variables are:
`{var1: "var1", var2: "newValue", var3: "otherVal", var4: "val4"}`
```
And remove: 
```
None of the modifications are propagated outside the ForEach Scope, including the creation of new variables.
```

Reply:
done